integration/src/test/java/edu/unc/lib/boxc/integration/web/access/EndpointIT.java

```diff
@@ -103,11 +104,42 @@ public void createDefaultObjects() throws Exception {
                 Map.of("title", "Folder Object", "readGroup", "everyone"));
     }
 
-    public List<JsonNode> getMetadataFromResponse(CloseableHttpResponse response) throws IOException {
+    public List<String> createDatedObjects() throws Exception {
```

Comment:
Were these creation methods moved up to the parent class in anticipation that they'll be needed by other tests later?

Reply:
I thought it'd be more consistent to have the object creation methods in one place, but I suppose we could put the more specialized ones in the SearchEndpointIT